products/revenue_analytics/frontend/revenueAnalyticsLogic.ts

```diff
@@ -303,6 +325,11 @@ export const revenueAnalyticsLogic = kea<revenueAnalyticsLogicType>([
             if (revenue_group_by && revenue_group_by !== values.grossRevenueGroupBy) {
                 actions.setGrossRevenueGroupBy(revenue_group_by)
             }
+
+            const parsedFilters = isRevenueAnalyticsPropertyFilters(filters) ? filters : undefined
+            if (parsedFilters && !objectsEqual(parsedFilters, values.revenueAnalyticsFilter)) {
+                actions.setRevenueAnalyticsFilters(parsedFilters)
+            }
```

Comment:
Copied straight from Web Analytics. This means people cannot set up arbitrary events. The backend is also resistant to this, it wont accept invalid filters.